e2e/benchmarks/local-benchmark/index.js

```diff
@@ -28,7 +28,8 @@ const BACKEND_FLAGS_MAP = {
     'WEBGL_FORCE_F16_TEXTURES', 'WEBGL_RENDER_FLOAT32_CAPABLE',
     'WEBGL_FLUSH_THRESHOLD', 'WEBGL_PACK_DEPTHWISECONV',
     'CHECK_COMPUTATION_FOR_ERRORS'
-  ]
+  ],
+  webgpu: ['CHECK_COMPUTATION_FOR_ERRORS', 'WEBGPU_DEFERRED_SUBMIT_BATCH_SIZE'],
```

Comment:
'CHECK_COMPUTATION_FOR_ERRORS' is not needed for webgpu. And only add `webgpu: ['CHECK_COMPUTATION_FOR_ERRORS', 'WEBGPU_DEFERRED_SUBMIT_BATCH_SIZE'` when `isBackendSupported('webgpu')` is true.